subsys/fb/cfb.c

```diff
@@ -141,11 +198,11 @@ static int cfb_reverse_bytes(const struct char_framebuffer *fb)
 
 	for (size_t i = 0; i < fb->x_res * fb->y_res / 8U; i++) {
 		fb->buf[i] = (fb->buf[i] & 0xf0) >> 4 |
-			      (fb->buf[i] & 0x0f) << 4;
+			     (fb->buf[i] & 0x0f) << 4;
```

Comment:
Please make all style fixes in separate commit.

Reply:
ok